adb_graphics/utils.py

```diff
@@ -246,34 +249,6 @@ def uniq_wgrib2_list(inlist):
 
     return uniq_list
 
-def zip_products(fhr, workdir, zipfiles):
-
-    ''' Spin up a subprocess to zip all the product files into the staged zip files.
-
-    Input:
-
-        fhr         integer forecast hour
-        workdir     path to the product files
-        zipfiles    dictionary of tile keys, and zip directory values.
-
-    Output:
-        None
-    '''
-
-    for tile, zipf in zipfiles.items():
-        if tile == 'skewt_csv':
-            file_tmpl = f'*.skewt.*_f{fhr:03d}.csv'
-        else:
-            file_tmpl = f'*_{tile}_*{fhr:02d}.png'
-        product_files = glob.glob(os.path.join(workdir, file_tmpl))
-        if product_files:
-            zip_proc = Process(group=None,
-                               target=create_zip,
-                               args=(product_files, zipf),
-                               )
-            zip_proc.start()
-            zip_proc.join()
-
```

Comment:
Just needed to move this down to be able to use the timer decorator, and because it was out of alphabetical order. I didn't make a change to the contents.